lib/tasks/delete_missed_reservations.rake

```diff
@@ -6,7 +6,8 @@ task delete_missed_reservations: :environment do
   unless AppConfig.first.blank? || AppConfig.first.res_exp_time.blank?
     time = AppConfig.first.res_exp_time
     missed_reservations = Reservation.where(
-      'checked_out IS NULL and due_date < ?', Date.current - time.days)
```

Comment:
~~Won't this only delete past reservations that weren't processed requests? In that case, *all* requests will just pile up in our database. We might want to approach this by dealing with the mailer task.~~